spec/support/matchers/accessibility.rb

```diff
@@ -112,8 +130,24 @@ def descriptors(element)
 end
 
 RSpec::Matchers.define :have_name do |name|
+  match { |element| AccessibleName.new(page:).computed_name(element) == name }
+
+  failure_message do |element|
+    <<-STR.squish
+      Expected element would have computed name "#{name}".
+      Found #{computed_name(element)}.
+    STR
+  end
+end
+
+class AccessibleName
```

Comment:
Could / should probably move this elsewhere and add tests for it? Unclear if that makes sense and where would be best for a utility only used in specs.

Reply:
I think this is fine here. How would this differ from the rest of the file?

Reply:
My main concern is if we'd want to have a convention of one-class-per-file, and files containing a class _only_ containing that class. It'd also be nice to have some test coverage for it, but we don't typically enforce test coverage for test helpers.

Reply:
You could break it out as accessibility_be_logically_grouped.rb or something like that. 
I wonder if the other classes in that test helper should be split off also?

Reply:
I think I'll keep this as-is for now, but I do like the idea of maybe creating a new folder within `helpers` for accessibility helpers, which would be more scalable and let us do one-class/matcher-per-file.